databases/week2/Creating a database.sql

```diff
@@ -0,0 +1,38 @@
+-- creating database --
+
+CREATE DATABASE Cake_Bakery;
+
+USE Cake_Bakery;
+
+SET NAMES utf8mb4;
+
+  
+CREATE TABLE `bakery_offices` (
+  `offices_id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `location` varchar(255) NOT NULL,
+  `created` datetime  NOT NULL
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `bakery_office_staffs` (
+`staff_id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`name` varchar(255) NOT NULL,
+`age` int(10) unsigned NOT NULL,
+`position_id` int(10) unsigned NOT NULL,
```

Comment:
is this a foreign key?

Reply:
But , I do not have any constraints on this position_id. So it is not a foreign key. :)

Reply:
What does `position_id` refer to? I am just wondering

Reply:
You need a `positions` table if you like to do it elegantly